netbox/utilities/testing/api.py

```diff
@@ -440,13 +440,8 @@ def _get_graphql_base_name(self):
             base_name = self.model._meta.verbose_name.lower().replace(' ', '_')
             return getattr(self, 'graphql_base_name', base_name)
 
-        def _build_query(self, name, **filters):
+        def _build_query_with_filter(self, name, filter_string):
```

Comment:
Could you add docstrings for these three methods please? They all look very similar and it's not immediately clear how they differ.

- `_build_query_with_filter()`
- `_build_filtered_query()`
- `_build_query()`

Reply:
done